dali/operators/decoder/audio/utils_test.cc

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
The file name is misleading - neither does it test some "utils" (it tests the decoder) nor does it contain any utilities itself (it contains standalone test cases).
I suggest either moving the contents to decoder tests or adding a new file llike "decoder_advanced_tests" or similar.